src/database.c

```diff
@@ -97,12 +101,16 @@ log_database_init(ProfAccount* account)
     }
 
     char* err_msg;
+
+    // ChatLogs Table
+    // Contains visual representation of chat logs
```

Comment:
Not sure what `visual representation of chat logs` should exactly mean..

Reply:
By "visual representation" I mean the chat messages you see when you open a chat window (i.e. corrected version)

Reply:
Line removed